src/pages/design/icons/index.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Icon Overview
+---
+
+import { graphql } from 'gatsby';
+
+import IconShowcase from '../../../examples/design/icons/IconShowcase.tsx';
+import IconShowcaseCode from '!!raw-loader!../../../examples/design/icons/IconShowcase.tsx';
+
+<CodeExample code={IconShowcaseCode}>
+  <IconShowcase />
+</CodeExample>
+
+### Usage
+
+Garden’s icon library is implemented using SVG. Icons come two sizes: 12px and 16px.
+The 16px icons can also be doubled to create 32px icons.
+
+#### Styles
```

Comment:
@bethanybeech What are your thoughts on making Styles an H3 and have it moved below 32px? The Usage description makes it read like we're going to just talk about sizes so let that come first and then Styles so the description flows "Icons also come in two styles..."

Also, thinking on this further, do you think 12px and 16px, 32px, and Styles should be H4s nesting under the H3 Usage?

Reply:
Yep, I think something may have gotten mixed up here. In my comp, 12px and 16px, 32px, and Styles are all nested under the H3 Usage.